.gitignore

```diff
@@ -113,6 +113,7 @@ build/
 # Test reports
 coverage/
 playwright-report/
```

Comment:
why is `client/playwright` not ignored? This is because the playwright output files were / may have been moved from playwright-report to playwright. Can we revert this change without breaking anything. Otherwise, we need to update the .gitignore.

Reply:
It gets created for code on `feature/distributed-demo` branch.

Reply:
The `.gitignore` has been updated in the latest commit on `feature/distributed-demo`. So this is taken care of.